### PR TITLE
Eng 2165

### PR DESCRIPTION
**Pull request by aecook**

### CODE REVIEW

File: modules.json [hunk 1]
```diff
@@ -34,5 +34,6 @@
     "tripleliftBidAdapter",
     "trustxBidAdapter",
     "undertoneBidAdapter",
-    "unrulyBidAdapter"
+    "unrulyBidAdapter",
+    "ffaBidAdapter"
```
Comment (dlr1967) [inline modules.json]: Should we put ffa in alphabetical.

Comment (aecook) [reply modules.json]: Updated to be alphabetical

File: modules/ffaBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+   * @return {Bid[]} An array of bids which were nested inside the server.
+   */
+  interpretResponse: function(serverResponse, bidRequest) {
+    // grab some vars
+    const bidResponses = [], floors = serverResponse.body.networkFloorMap, bids = bidRequest.data.validBidRequests;
+    // loop through the bids
+    bids.forEach((bid) => {
+      // set some vars
+      let cpm = 0;
+      // if there is a value for  0 (`ffa`) use that
```
[the remark this answers is not in the document]
Comment (dlr1967) [reply modules/ffaBidAdapter.js]: ffa has been added to the network table, and it is id 68

**Comment by aecook** [comment]
@daniel-austin not removing the logic, in fact make use of it in case there is no floor value for the FAA present in networkFloorMap...

**Pull request merged**